### ISSUE TITLE
Timing plots

### ISSUE THREAD

**Issue by mikebentley15** [PRUNERS/FLiT#75]
In the paper submission, there is a timing plot that shows the reproducible points and is sorted by execution time.  This functionality should be part of the repository, and hopefully part of the flit analyze command.

**Comment by ganeshutah** [comment]
Mike,

I've been seeing all your messages. A lot is going on!  The plot generation
automation would be great.

Best,

Ganesh

**Comment by mikebentley15** [comment]
Thanks Ganesh.  I like to use the GitHub issues as a task list.  It helps
me keep organized.  And I can create a new issue when I think of something
that I don't have time to work on at the moment - as kind of a back burner.

Sorry about the spam to your email

**Comment by mikebentley15** [comment]
Implementation is finished, but needs the following before we can close it:

- Documentation
- Unit tests (as much as is reasonable)
